src/IssueViz.Security.UnitTests/Taint/TaintIssuesSynchronizerTests.cs

```diff
@@ -306,34 +338,39 @@ public async Task SynchronizeWithServer_StandaloneMode_UIContextIsCleared()
             var monitorMock = CreateMonitorSelectionMock(cookie);
             var toolWindowServiceMock = new Mock<IToolWindowService>();
 
-            var testSubject = CreateTestSubject(mode: SonarLintMode.Standalone,
+            var testSubject = CreateTestSubject(
+                bindingConfig: BindingConfig_Standalone,
                 sonarService: sonarServiceMock.Object,
                 vsMonitor: monitorMock.Object);
 
             await testSubject.SynchronizeWithServer();
 
-            sonarServiceMock.Invocations.Count().Should().Be(0);
+            sonarServiceMock.Invocations.Should().HaveCount(0);
             CheckUIContextUpdated(monitorMock, cookie, 0);
-            toolWindowServiceMock.Invocations.Count.Should().Be(0);
+            toolWindowServiceMock.Invocations.Should().HaveCount(0);
         }
 
         [TestMethod]
-        public async Task SynchronizeWithServer_ConnectedMode_UsesExpectedBranch()
+        [DataRow(SonarLintMode.Connected)]
+        [DataRow(SonarLintMode.LegacyConnected)]
```

Comment:
Miscellaneous: most of the other connected mode test are data-driven to test both connected modes, so I made this one do the same.